ksqldb-streams/src/test/java/io/confluent/ksql/execution/streams/SourceBuilderTest.java

```diff
@@ -580,15 +584,20 @@ public void shouldBuildTableWithCorrectStoreName() {
   }
 
   @SuppressWarnings("unchecked")
-  private <K> ValueTransformerWithKey<K, GenericRow, GenericRow> getTransformerFromTableSource(
+  private <K> List<ValueTransformerWithKey<K, GenericRow, GenericRow>> getTransformersFromTableSource(
```

Comment:
Not sure if this is the ideal way of applying multiple transformers? It's extensible, but I doubt we'll be using more transformers anytime soon. I don't think it reads _too_ badly, but it's definitely worse than before.

Reply:
I think it's fine. This is just for these unit tests anyway.